django_socio_grpc/proto_serializers.py

```diff
@@ -40,6 +42,8 @@ def many_init(cls, *args, **kwargs):
         child_serializer = cls(*args, **kwargs)
         list_kwargs = {
             "child": child_serializer,
+            "message": kwargs.get("message", None),
```

Comment:
create a new variable LIST_PROTO_SERIALIZER_KWARGS that extends LIST_SERIALIZER_KWARGS with this two fields